test/run.js

```diff
@@ -50,9 +50,14 @@ async function runMatchingTests(state) {
 
       console.log(`[${elapsedTime(state)}] TestStart:${test} target:${target}`);
       const exampleRecordingId = await getExample(state, example, target);
-      console.log(
-        `[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`
-      );
+      if (exampleRecordingId) {
+        console.log(
+          `[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`
+        );
+      } else {
+        console.log(`::error ::Failure Failed to record example ${example}`);
```

Comment:
Is there any structure to our logging format here? Curious why "TestStart" and "TestExample" vs "::error ::"
```js
console.log(`[${elapsedTime(state)}] TestStart:${test} target:${target}`);
console.log(`[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`);
console.log(`::error ::Failure Failed to record example ${example}`);
```